includes/admin/views/builder/question.php

```diff
@@ -171,6 +173,18 @@
 				<# } #>
 			</div>
 
+			<# if ( 'yes' === data.collection.parent.attributes.question_bank ) { #>
+			<div class="llms-settings-row">
+				<div class="llms-editable-toggle-group">
+					<label class="llms-switch">
+						<span class="llms-label"><?php esc_html_e( 'Include in Questions Bank', 'lifterlms' ); ?></span>
+						<input type="checkbox" name="include_bank_variations"<# if ( 'yes' === data.get( 'include_bank_variations' ) ) { print( ' checked' ) } #>>
+						<div class="llms-switch-slider"></div>
+					</label>
+				</div>
+			</div>
+			<# } #>
+
```

Comment:
I would like to keep the logic in the advanced add-on as much as we can.

what if we have a `<?php do_action( 'llms_builder_question_after_clarifications' ); ?>` that we can leverage from the add-on?
Or even better, can't we just leverage the existing `llms_builder_question_after_features` action hook?

Reply:
Done. moved the code to advance quizzes.